backend/de.metas.salescandidate.base/src/main/sql/postgresql/system/45-de.metas.salescandidate/5635530_sys_gh12840_drop_AD_RelationType_and_AD_Scheduler_for_C_OLCand_Process.sql

```diff
@@ -0,0 +1,10 @@
+-- 2022-04-15T08:51:29.428Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+DELETE FROM AD_RelationType WHERE AD_RelationType_ID=540101
+;
+
+-- 2022-04-15T08:52:00.194Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+DELETE FROM AD_Scheduler WHERE AD_Scheduler_ID=550015
```

Comment:
don't we also need to delete AD_scheduler-log records?

Reply:
deleted also the `AD_scheduler-log` records